### PR TITLE
Flags correction for windows

### LINKED ISSUE #804: Wrong openmp flag on Windows
We're testing `-fopenmp` for Windows, but I think the correct flag for visual studio 2015 is just `/openmp` : https://msdn.microsoft.com/en-us/library/tt15eb9t.aspx

### PR DESCRIPTION
**Pull request by skoudoro**
The goal is this PR is to setup correctly flags following the operating system. Moreover This should resolve #804.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -52,56 +54,54 @@
...
-    ('dipy.align.vector_fields', [], 'c'),
-    ('dipy.align.sumsqdiff', [], 'c'),
-    ('dipy.align.expectmax', [], 'c'),
-    ('dipy.align.crosscorr', [], 'c'),
-    ('dipy.align.bundlemin', [], 'c'),
-    ('dipy.align.transforms', [], 'c'),
-    ('dipy.align.parzenhist', [], 'c'),
-    ('dipy.utils.omp', [], 'c')):
-
+        ('dipy.reconst.peak_direction_getter', [], 'c'),
```
Comment (Garyfallidis) [inline setup.py]: too much indentation

File: setup.py [hunk 2]
```diff
@@ -118,11 +118,21 @@
 simple_test_c = """int main(int argc, char** argv) { return(0); }"""
 omp_test_c = """#include <omp.h>
 int main(int argc, char** argv) { return(0); }"""
-extbuilder = add_flag_checking(
-    build_ext, [
-        [['/arch:SSE2'], [], simple_test_c, 'USING_VC_SSE2'],
-        [['-msse2', '-mfpmath=sse'], [], simple_test_c, 'USING_GCC_SSE2'],
-        [['-fopenmp'], ['-fopenmp'], omp_test_c, 'HAVE_OPENMP']], 'dipy')
+
+msc_flag_defines = [[['/openmp'], [], omp_test_c, 'HAVE_VC_OPENMP'],
```
Comment (Garyfallidis) [inline setup.py]: I think msc supports SSE2

File: setup.py [hunk 3]
```diff
@@ -118,11 +118,21 @@
...
+                    ]
+gcc_flag_defines = [[['-msse2', '-mfpmath=sse'], [], simple_test_c, 'USING_GCC_SSE2'],
+                    [['-fopenmp'], ['-fopenmp'], omp_test_c, 'HAVE_OPENMP'],
+                    ]
+
+# Test if it is a 32 bits version
+if not sys.maxsize > 2 ** 32:
+    # This flag is needed only on 32 bits
+    msc_flag_defines += [[['/arch:SSE2'], [], simple_test_c, 'USING_VC_SSE2'], ]
```
Comment (Garyfallidis) [inline setup.py]: Ah I see. It is here.

File: setup.py [hunk 4]
```diff
@@ -164,65 +174,65 @@ def main(**extra_args):
...
           # The package_data spec has no effect for me (on python 2.6) -- even
           # changing to data_files doesn't get this stuff included in the source
           # distribution -- not sure if it has something to do with the magic
           # above, but distutils is surely the worst piece of code in all of
           # python -- duplicating things into MANIFEST.in but this is admittedly
           # only a workaround to get things started -- not a solution
-          package_data = {'dipy':
-                          [pjoin('data', 'files', '*')
-                          ]},
+          package_data={'dipy':
```
Comment (Garyfallidis) [inline setup.py]: Pep 8. Spaces around operators.

Comment (Garyfallidis) [inline setup.py]: Correct everywhere please.

Comment (skoudoro) [inline setup.py]: As you can see [here](https://www.python.org/dev/peps/pep-0008/#other-recommendations), PEP 8 recommend not having spaces around = in a keyword argument or a default parameter value. I correct it everywhere for this reason. Are you sure it needs a revert ?

**Comment by Garyfallidis** [comment]
OpenMP in Windows done! OSX is next :)

**Comment by Garyfallidis** [comment]
Thanks @skoudoro!

**Pull request merged**